ci/deploy_app.yaml

```diff
@@ -41,10 +40,8 @@ run:
 
       gcloud container clusters get-credentials survey-runner --region "${REGION}" --project "${PROJECT_ID}"
 
-      helm init --client-only
-      helm  plugin install https://github.com/rimusz/helm-tiller
-
       cd eq-questionnaire-runner
       ./k8s/deploy_app.sh
 
+      kubectl rollout restart deployment.v1.apps/runner
```

Comment:
@ajmaddaford @JamesGardiner 

Added this to restart the deployment on every execution of the task. Hence, it will re-pull the image even if the image tag has not changed. This behaves the same as if the image tag were to change.